packages/data/tests/unit/services/navi-metadata-test.ts

```diff
@@ -0,0 +1,267 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import { TestContext } from 'ember-test-helpers';
+import config from 'ember-get-config';
+import NaviMetadataService from 'navi-data/services/navi-metadata';
+import TableMetadataModel from 'navi-data/models/metadata/table';
+//@ts-ignore
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import ElideTwoScenario from 'dummy/mirage/scenarios/elide-two';
+import DimensionMetadataModel from 'navi-data/models/metadata/dimension';
+import MetricMetadataModel from 'navi-data/models/metadata/metric';
+import TimeDimensionMetadataModel from 'navi-data/models/metadata/time-dimension';
+import ColumnFunctionMetadataModel from 'navi-data/models/metadata/column-function';
+
+interface Context extends TestContext {
+  server: TODO;
```

Comment:
I think we can use this type
```js
import { Server } from 'pretender';
```

Reply:
Mirage's `server` is not exactly the same as pretender's `server`. I  believe Mirage's `server.pretender` is the same as that import.